### PR TITLE
get/import: they don't work with any DVC project, only DVC repos (as well as Git repos)

### PR DESCRIPTION
**Pull request by jorgeorpinel**
At least on v `0.82.6`. I created a `--no-scm` project and tried to `get` and `import` from it to other locations, getting:

```
ERROR: failed to get 'data' from '/Users/drbidibombom/DVC-repos/no-scm-aux' - Failed to clone repo '/Users/drbidibombom/DVC-repos/no-scm-aux' to '/var/folders/_c/3mt_xn_d4xl2ddsx2m98h_r40000gn/T/tmpy1o07ch5dvc-clone': Cmd('git') failed due to: exit code(128)
  cmdline: git clone --no-single-branch -v /Users/drbidibombom/DVC-repos/no-scm-aux /var/folders/_c/3mt_xn_d4xl2ddsx2m98h_r40000gn/T/tmpy1o07ch5dvc-clone
  stderr: 'fatal: repository '/Users/drbidibombom/DVC-repos/no-scm-aux' does not exist
'


Having any troubles? Hit us up at https://dvc.org/support, we are always happy to help!
```

---

* [x] ❗ Have you followed the guidelines in the [Contributing to DVC](https://dvc.org/doc/user-guide/contributing/core) list?

* [x] 📖 Check this box if this PR **does not** require [documentation](https://dvc.org/doc) updates, or if it does **and** you have created a separate PR in [dvc.org](https://github.com/iterative/dvc.org) with such updates (or at least opened an issue about it in that repo). Please link below to your PR (or issue) in the [dvc.org](https://github.com/iterative/dvc.org) repo.

  Docs update in iterative/dvc.org/pull/931 & iterative/dvc.org/pull/978

* [x] ❌ Have you checked DeepSource, CodeClimate, and other sanity checks below? We consider their findings recommendatory and don't expect everything to be addressed. Please review them carefully and fix those that actually improve code or fix bugs.

Thank you for the contribution - we'll try to review it as soon as possible. 🙏

### CODE REVIEW

File: dvc/command/get.py [hunk 1]
```diff
@@ -51,9 +51,7 @@ def _get_file_from_repo(self):
 
 
 def add_parser(subparsers, parent_parser):
-    GET_HELP = (
-        "Download file or directory from any DVC project or Git repository."
-    )
+    GET_HELP = "Download file or directory from any DVC or Git repository."
```
Comment (efiop) [inline dvc/command/get.py]: Feels like `DVC or Git` has the same meaning as `DVC project or Git`. Maybe `Git repository w/ or w/o DVC` or something like that?

Comment (jorgeorpinel) [reply dvc/command/get.py]: A "DVC project" doesn't have to be in a Git repo but these commands require a Git repo.

I realize it's tricky wording but I'd prefer to mention DVC before Git. Maybe just "DVC repository or Git repo"? It's kind of repetitive though...

Comment (efiop) [reply dvc/command/get.py]: Well, but that DVC repo has to be a git repo too, that is why I'm wondering if we could clarify that.

Comment (shcheklein) [reply dvc/command/get.py]: `Git repository w/ or w/o DVC` does not sound right to me. Feels like we deal with Git in the first place. While we should emphasize that it's about DVC controlled files.

How about: `Download a Git or DVC controlled file or directory`?

Comment (shcheklein) [reply dvc/command/get.py]: I think @jorgeorpinel point is that relying on DVC repository (has to be Git repo) vs DVC project (can be Git or not Git) is very hard to understand, very hard to maintain in docs. Phrases like `DVC project or Git repository` probably won't mean much for users.

Comment (jorgeorpinel) [reply dvc/command/get.py]: Hi guys, getting back to this finally.

> relying on DVC repository (has to be Git repo) vs DVC project (can be Git or not Git) is very hard to understand

Agreed.

> How about: Download a Git or DVC controlled file or directory?

Good option, updated to "tracked by DVC or by Git" for both commands in this PR and docs in iterative/dvc.org/pull/978

Comment (jorgeorpinel) [reply dvc/command/get.py]: Resolving this, but please feel free to participate in this related discussion Ruslan: https://github.com/iterative/dvc.org/pull/931#pullrequestreview-354892810

Thanks!

**Comment by jorgeorpinel** [comment]
- [ ] This is blocking jorgeorpinel/dvc/pull/1

**Comment by efiop** [comment]
Oops, sorry for the delay @jorgeorpinel !

**Comment by jorgeorpinel** [comment]
Ah no problem, I didn't mean to rush you @efiop, I just wanted to put a checkbox there for myself not to forget about my next PR. Thanks though!

**Comment by efiop** [comment]
@jorgeorpinel Please feel free to ping me (e.g. in PMs) if I don't respond to something in one day, I do sometimes miss/lose some notifications.

**Comment by jorgeorpinel** [comment]
Thanks

**Pull request merged**